alphastats/gui/pages/02_Import Data.py

```diff
@@ -61,8 +79,7 @@
 options = [default_select_option] + list(SOFTWARE_OPTIONS.keys())
 
 software = st.selectbox(
-    "Select your Proteomics Software",
-    options=options,
+    "Select your Proteomics Software", options=options, key="_software"
```

Comment:
Do I get it right, that you want to use underscores to mark widget keys, as opposed to manually set session_states like dataset?